pkg/tinyvcdiff/contrib/tinyvcdiff_vfs/vfs.c

```diff
@@ -0,0 +1,66 @@
+#include "vcdiff_vfs.h"
+#include "errno.h"
+
+static int read (void *dev, uint8_t *dest, size_t offset, size_t len)
+{
+    int rc;
+    vcdiff_vfs_t *vfs = (vcdiff_vfs_t *) dev;
```

Comment:
```suggestion
    vcdiff_vfs_t *vfs = dev;
```

no need to cast the `void *`